examples/endian_swapper/tests/Makefile

```diff
@@ -30,22 +30,20 @@
 # Default to verilog
 TOPLEVEL_LANG ?= verilog
 
-WPWD=$(shell sh -c 'pwd -W')
 PWD=$(shell pwd)
 
 ifeq ($(OS),Msys)
 WPWD=$(shell sh -c 'pwd -W')
-PYTHONPATH := $(PWD)/../cosim;$(PYTHONPATH)
+PYTHONPATH := $(WPWD)/../cosim;$(PYTHONPATH)
```

Comment:
Are you sure this windows special case is actually necessary?

My understanding was that msys automatically translated colon separated paths.

Reply:
Not sure so did not touch but can check.

Reply:
I have tried to make this work on Windows but this is not so easy (multiple quite involved issues). So whatever we do here does not make any difference.

Reply:
I can try to come back to this and fix this example with pybind11 or setup tools at some point.